tests/integration/long/test_ssl.py

```diff
@@ -41,12 +49,21 @@ def setup_module():
     ccm_cluster.stop()
 
     # Fetch the absolute path to the keystore for ccm.
-    abs_path_server_keystore_path = os.path.abspath(DEFAULT_SERVER_KEYSTORE_PATH)
+    abs_path_server_keystore_path = os.path.abspath(SERVER_KEYSTORE_PATH)
 
     # Configure ccm to use ssl.
+
     config_options = {'client_encryption_options': {'enabled': True,
                                                     'keystore': abs_path_server_keystore_path,
                                                     'keystore_password': DEFAULT_PASSWORD}}
+
+    if(client_auth):
+        abs_path_server_truststore_path = os.path.abspath(SERVER_TRUSTSTORE_PATH)
+        client_encyrption_options = config_options['client_encryption_options']
+        client_encyrption_options['require_client_auth'] = True
+        client_encyrption_options['truststore'] = abs_path_server_truststore_path
+        client_encyrption_options['truststore_password'] = DEFAULT_PASSWORD
+
     ccm_cluster.set_configuration_options(config_options)
     ccm_cluster.start(wait_for_binary_proto=True, wait_other_notice=True)
```

Comment:
I've ben running in this manner. "CASSANDRA_VERSION=2.2.0-rc2 nosetests -s -v tests/integration/long/test_ssl.py". I've also tried with C* version 2.1.8, with no issues. I'm not sure this test can be run against a standing C* cluster as it needs to restart the CCM cluster with custom yaml configs for the various certs/keys.